django/applications/catmaid/templates/catmaid/index.html

```diff
@@ -50,10 +50,6 @@
         {% endif %}
     {% endfor %}
 
-    {% for f in NON_COMPRESSED_FILES %}
-      <script type="application/javascript" src="{{ STATIC_URL }}{{ f }}"></script>
-    {% endfor %}
-
```

Comment:
This we still need. Only the neuroglancer-specific part should be removed, i.e. the version above is fine as is.